src/assets/scripts/client/math/core.js

```diff
@@ -84,12 +84,29 @@ export function s(i) {
  */
 export function isWithin(value, limit1, limit2) {
     if (limit1 > limit2) {
-        const oldLimit1 = limit1;
-        limit1 = limit2;
-        limit2 = oldLimit1;
+        [limit2, limit1] = [limit1, limit2];
```

Comment:
hmm, I don't remember seeing this one earlier.  what exactly are you doing here?

Are you creating an array (right side) but immediately destructuring to different assignments?  Either way, this is really confusing.  Is there a more obvious way to do what's needed here?

Clever solutions are fun, for sure, but they make things harder to maintain.  Simple is best 😉

Reply:
That's exactly what you said. The goal is to swap variables, and that's the cleaner notation I've ever seen. I'm not usually fond of clever tricks either, but this is as explicit as we can get to swap vars.

We can restore it to what it was before, it was doing the job using a temporary variable `oldlimit1`:
```suggestion
        const oldLimit1 = limit1;
        limit1 = limit2;
        limit2 = oldLimit1;
```

Reply:
😌 

This is certainly more verbose, and perhaps not as fun, but this is much easier to read and understand quickly.  Thanks for changing this!